project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject trapperkeeper/lein-template "1.1.0"
+(defproject trapperkeeper/lein-template "1.3.0-SNAPSHOT"
```

Comment:
Hm, okay. I guess that makes sense. I would maybe worry about what happens if we make changes to the template other than just bumping the tk version, what do we bump the version to then? But version numbers are terrible and I don't want to think about them too much so I'm fine with whatever.